components/Header.js

```diff
@@ -38,12 +51,18 @@ class Header extends React.Component {
     };
 
     this.meta = [];
+
     for (const name in meta) {
       this.meta.push({
         name,
         content: meta[name],
       });
     }
+
+    this.title = title || 'Open Collective - open your finances to your community';
+    if (!this.title.match(/open collective/i)) {
+      this.title += ' - Open Collective';
```

Comment:
Better to avoid storing the title in `this` as any update to the props won't be reflected. For example in https://github.com/opencollective/opencollective-frontend/blob/80be17d5ed84855588bebce64b36f1d53760c2b9/pages/createOrder.js#L87, we initially set the page's title to `Contribute` then later include the collective name when loaded.